docs/guides/8-continuous-integration.md

```diff
@@ -0,0 +1,49 @@
+# Continuous Integration
+
+## GitHub Action
+
+Spectral has a pre-built [Spectral GitHub Action](https://github.com/stoplightio/spectral-action) which should speed up implementing Spectral in your GitHub repository.
+
+## CircleCI
+
+Users of CircleCI can take this entire file and save it as `.circleci/config.yml`, or move the `lint` job into their existing config.
+
+```yaml
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1
+
+jobs:
+  lint:
+    docker:
+      - image: cimg/node:15.1
+    steps:
+      # Checkout the code as the first step.
+      - checkout
+      # Create a folder for results to live in
+      - run: "[ -d lint-results ] || mkdir lint-results"
+      - run:
+          name: Run Spectral Lint
+          command:
+            npx @stoplight/spectral lint openapi.yaml
```

Comment:
Should we indicate that more than multiple files or glob patterns can be provided?

Reply:
My theory was that we arent teaching the CLI here, we are teaching how to CI.

Reply:
Agree with Phil. Lets keep it simple here.